provider/core/src/data_provider.rs

```diff
@@ -74,6 +74,23 @@ where
     }
 }
 
+/// A data provider that can determine whehter it can load a particular data identifier,
+/// potentially cheaper than performing the load.
```

Comment:
Issue: Should we add some discussion about the data needing to be built with RetainBaseLanguages or NoDeduplication?

Reply:
Hmm I think this discussion should be on LocaleFallbackProvider, and on the baked macros.

Reply:
Actually we probably want a correct can_load on the fallbacker that returns true for und, and then a non-trait can_load_skip_und that has the deduplication warning.